bazel/repository_locations.bzl

```diff
@@ -388,7 +405,7 @@ DEPENDENCY_REPOSITORIES_SPEC = dict(
         sha256 = "5fe543e8d007b9e7b729f3d6b3a5ee1f9b68d0eef5f6af1393745a4dcd472a98",
         strip_prefix = "zlib-ng-193d8fd7dfb7927facab7a3034daa27ad5b9df1c",
         urls = ["https://github.com/zlib-ng/zlib-ng/archive/193d8fd7dfb7927facab7a3034daa27ad5b9df1c.tar.gz"],
```

Comment:
`urls = ["https://github.com/zlib-ng/zlib-ng/archive/193d8fd7dfb7927facab7a3034daa27ad5b9df1c.tar.gz"],` to `urls = ["https://github.com/zlib-ng/zlib-ng/archive/{version}.tar.gz"],`